### PR TITLE
Add javadoc for CMV

### LINKED ISSUE #3: Documentation
Start writing documentation for the project. This must not be a final version of the README.md, but it should get things started. 

More information should be added on the go.

### PR DESCRIPTION
**Pull request by Berisha1999**
Fixes #3

### CODE REVIEW

File: src/main/java/com/group23/decide/LIC.java [hunk 1]
```diff
@@ -35,7 +35,8 @@ public static boolean condition0(int NUMPOINTS, Point[] POINTS, double LENGTH1)
     }
 
     /**
-     * Function that computes condition 1 of the LIC.
+     * Function that checks if there are at least a set of three consecutive points that cannot all be contained within a circle of 
+     * Radius1
```
Comment (medkamilmel) [inline src/main/java/com/group23/decide/LIC.java:39]: maybe write "circle or radius RADIUS1"

File: src/main/java/com/group23/decide/LIC.java [hunk 2]
```diff
@@ -530,8 +534,10 @@ public static boolean condition13(int NUMPOINTS, Point[] POINTS, double RADIUS1,
     }
 
     /**
-     * Function that computes condition 14 of the LIC.
-     *
+     * Function that checks if there exists at least on set of three consecutive points separated by exactly E_PTS and F_PTS points
+     * that are the vertices of a triangle with an area larger than AREA1 and that there exists a set of three consecutive points 
+     * separeated by exactly E_PTS and P_PTS that are the vertices of a triangle with an area less than AREA2. The set of points 
+     * do not have to be the same. Both conditions have to be met for the LIC to be true. If NUMPOINTS < 5 the condition is not met. 
      * @param ...
```
Comment (medkamilmel) [inline src/main/java/com/group23/decide/LIC.java:541]: missing the parameters javadoc

**Review by medkamilmel — CHANGES_REQUESTED** [review]

File: src/main/java/com/group23/decide/LIC.java (hunk 1 above)
Comment (Berisha1999) [reply src/main/java/com/group23/decide/LIC.java]: Yes will fix that

Comment (Berisha1999) [reply src/main/java/com/group23/decide/LIC.java]: I have changed that

File: src/main/java/com/group23/decide/LIC.java (hunk 2 above)
Comment (Berisha1999) [reply src/main/java/com/group23/decide/LIC.java]: This is now fixed

**Review by medkamilmel — APPROVED** [review]

**Pull request merged**